src/main/scala/com/advancedtelematic/director/daemon/DaemonBoot.scala

```diff
@@ -31,8 +31,8 @@ object DaemonBoot extends BootApp
     with MetricsSupport
     with DatabaseMetrics
     with MessageListenerSupport
-    with InfluxdbMetricsReporterSupport
     with AkkaHttpRequestMetrics
+    with CheckMigrations
```

Comment:
Shouldn't we `CheckMigrations` at `Boot` as well? We do it for campaigner and device-registry.

Reply:
Yes! Thanks